migrations/m190326_112703_xcoin_funding_descrption.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Class m190326_112703_xcoin_funding_descrption
+ */
+class m190326_112703_xcoin_funding_descrption extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $this->alterColumn('xcoin_funding', 'description', $this->string(255)->notNull());
```

Comment:
The description field was a text type, but for UI concerns malek said that we should restrict the length of the field for a better look&feel.

Reply:
ok, I would have left the DB column but only add a data validation for the length. this would make it easier to change and put less restrictions from the DB on the data. 
afaik there is no real speed improvement anymore if the column is a char with fixed length or any kind of text.

Reply:
You're right , i'll do it.

Reply:
Done .